thunderbird/MozillaThunderbird.spec

```diff
@@ -370,27 +368,19 @@ ac_add_options --disable-elf-hack
 %endif
 ac_add_options --with-system-nspr
 ac_add_options --with-system-nss
-%if %{localize}
-ac_add_options --with-l10n-base=$RPM_BUILD_DIR/l10n
-%endif
-#ac_add_options --with-system-jpeg    # libjpeg-turbo is used internally
-#ac_add_options --with-system-png     # doesn't work because of missing APNG support
 ac_add_options --with-system-zlib
+ac_add_options --with-l10n-base=$RPM_BUILD_DIR/l10n
```

Comment:
Since the localize-flag still exists and is used to compile (or not compile) the translations-packages, shouldn't it be kept here as well? To be able to build TB without any l10n-stuff, if need be?

Reply:
yes